draft-ietf-quic-transport.md

```diff
@@ -1065,6 +1066,13 @@ sending a packet with a number of 0x6b2d79 requires a packet number encoding
 with 14 bits or more; whereas the 30-bit packet number encoding is needed to
 send a packet with a number of 0x6bc107.
 
+An endpoint MUST discard packets with duplicate packet numbers.  After
+unprotecting a packet, the receiver MUST check that it has not already processed
+a packet with this PN and if so discard it. Duplicate suppression MUST happen
```

Comment:
The receiver isn't necessarily able to confirm that for all packet numbers.  I think the language needs to be that if the receiver MUST discard if it can't verify that it hasn't processed this packet number before.